Library/Homebrew/formula.rb

```diff
@@ -2133,6 +2133,7 @@ def to_hash
       "disable_date"             => disable_date,
       "disable_reason"           => disable_reason,
       "tap_git_head"             => tap_git_head,
+      "ruby_source_checksum"     => {},
```

Comment:
Why not just put the checksum value directly here? Are we expecting to have multiple checksums at some point in the future?

Reply:
Nah, was me being overly safe (e.g. if we had a need to use anything else but SHA256 then we can without breaking backwards compatibility). Though probably not much point to do so.

It was modeled after how `Checksum` class worked but I see that got removed at some point and I never realised.